<?xml version="1.0"?>
<sdf version="1.6">
  <model name="part_200g">
    <link name="link">
      <pose>0 0 0 0 0 0</pose>

      <inertial>
        <mass>0.2</mass>
        <pose>0 0 0 0 0 0</pose>
        <inertia>
          <ixx>1.333e-5</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>1.333e-5</iyy>
          <iyz>0</iyz>
          <izz>4.0e-5</izz>
        </inertia>
      </inertial>

      <collision name="collision">
        <geometry>
          <cylinder>
            <radius>0.02</radius>
            <length>0.02</length>
          </cylinder>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>0.8</mu>
              <mu2>0.8</mu2>
            </ode>
          </friction>
          <contact>
            <ode>
              <kp>1000000.0</kp>
              <kd>100.0</kd>
              <max_vel>1.0</max_vel>
              <min_depth>0.001</min_depth>
            </ode>
          </contact>
        </surface>
      </collision>

      <visual name="visual">
        <geometry>
          <cylinder>
            <radius>0.02</radius>
            <length>0.02</length>
          </cylinder>
        </geometry>
        <material>
          <ambient>0.5 0.5 0.65 1</ambient>
          <diffuse>0.5 0.5 0.65 1</diffuse>
          <specular>0.5 0.5 0.5 1</specular>
        </material>
      </visual>
    </link>
  </model>
</sdf>
